AustinKellar/PowerOn-Prototype
# Replace art with physics

Goal: Replace the placeholder graphics with the art assets that our artists designed.

What I changed:
Swapped the player, tree, and branch graphics (which we rectangles) with the pngs the artists provided.
Added the artists' background (it's currently static).
Adjusted the sap bar to: make it bigger, center it at the top of the screen, change the direction so that it depletes from right to left, and make the fill color yellow (like the sap that the player collects).
Adjusted player physics by increasing the jump force to 17 and the gravity to 3.5.
Increased tree branch growth rate to .31.
Deleted comments that contained outdated code, added punctuation to comments, and revised one of the comments that contained attribution credit to ensure that the source I attributed was more relevant to the 2D raycasts we currently use instead of the outdated 3D raycasts we'd originally used.

How to Test:
Load up the game in the new branch
Ensure that the new sprites mesh well together.
Test the player movement with branch rate increase. Do these changes in speed help the game or hinder it?

Note:
Currently the sprites are low-quality pngs, but since the .ai files are in the Google Drive we can re-export them at a higher DPI. I haven't done so yet.
